### PR TITLE
Add syntax gram of `load stats`  (#3362)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #3362 to release-3.0

---

### What is changed, added or deleted? (Required)

Add syntax gram of `load stats`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

- This PR is translated from:
- Other reference link(s): cherry-picked from https://github.com/pingcap/docs-cn/pull/3340/

### CODE REVIEW

File: sql-statements/sql-statement-load-stats.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (yikeke) [inline sql-statements/sql-statement-load-stats.md:1]: Please add this file to TOC.md and fix lint. @qw4990

Comment (qw4990) [reply sql-statements/sql-statement-load-stats.md:1]: Updated. PTAL.

File: sql-statements/sql-statement-load-stats.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+title: LOAD STATS
+summary: TiDB 数据库中 LOAD STATS 的使用概况。
+category: reference
+aliases: ['/docs-cn/v3.0/reference/sql/statements/load-stats/']
```
Comment (yikeke) [inline sql-statements/sql-statement-load-stats.md:5]:
```suggestion
```

**Pull request merged**